port: /dev/ttyUSB0
HARDWARE_SYSTEM: RS3400K
FREQUENCY_START: 24.00e9
FREQUENCY_STOP: 25.50e9
FREQUENCY_POINTS: 1501
SWEEP_TIME: 0.3
SWEEP_MEASURE: ON
SWEEP_NUMBERS: 1
TRIGGER_SOURCE: IMMEDIATE